src/componenta/personalInfo/personalInfo.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './person.scss';
+import { MarriedOrNot } from './MarriedFun';
+
+export function PersonalInfo(props) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props.person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age
+        ? (
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        ) : null}
+      <p className="Person__partner">
+        {MarriedOrNot(
+          sex,
+          isMarried,
+          partnerName,
+        )}
```

Comment:
Will be better to create separate variable and store here the result of this function
And use this variable in jsx